test/JDBC/expected/BABEL-TABLEOPTIONS-vu-prepare.out

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE babel_tableoptions_vu_prepare_t1 (a INT, CONSTRAINT pk PRIMARY KEY CLUSTERED (a ASC)) WITH
```

Comment:
It would be nice to have a test case for a view.

Reply:
For views, Babelfish only supports `SCHEMABINDING` option, which does not get stored in reloptions so that's why I haven't added any test cases for views.